docs/book/orders/map.rst.inc

```diff
@@ -1,16 +1,12 @@
-.. raw:: html
-
-    <ul class="simple">
-    <li><p><a class="reference internal" href="orders/orders.html"><span class="doc">Orders</span></a></p></li>
-    <li><p><a class="reference internal" href="orders/cart-flow.html"><span class="doc">Cart flow</span></a></p></li>
-    <li><p><a class="reference internal" href="orders/taxation.html"><span class="doc">Taxation</span></a></p></li>
-    <li><p><a class="reference internal" href="orders/adjustments.html"><span class="doc">Adjustments</span></a></p></li>
-    <li><p><a class="reference internal" href="orders/promotions.html"><span class="doc">Promotions</span></a></p></li>
-    <li><p><a class="reference internal" href="orders/coupons.html"><span class="doc">Coupons</span></a></p></li>
-    <li><p><a class="reference internal" href="orders/payments.html"><span class="doc">Payments</span></a></p></li>
-    <li><p><a class="reference internal" href="orders/invoices.html"><span class="doc">Invoices</span></a></p></li>
-    <li><p><a class="reference internal" href="orders/shipments.html"><span class="doc">Shipments</span></a></p></li>
-    <li><p><a class="reference internal" href="orders/checkout.html"><span class="doc">Checkout</span></a></p></li>
-    <li><p><a class="reference internal plus-feature" href="orders/returns.html"><span class="doc">Returns</span></a></p></li>
-    <li><p><a class="reference internal" href="orders/refunds.html"><span class="doc">Refunds</span></a></p></li>
-    </ul>
+* :doc:`/book/orders/orders`
+* :doc:`/book/orders/cart-flow`
```

Comment:
So what convention do we have? `-` or `_`?

Reply:
`_` in general, but there are some mistakes in the filenames made previously.